documentation/modules/oauth/proc-oauth-authorization-broker-config.adoc

```diff
@@ -0,0 +1,95 @@
+
+Typically, groups are used to match users based on organizational departments or geographical locations.
+And roles are used to match users based on their function.
+
+With {oauth-server}, you can store users and groups in LDAP, whereas clients and roles cannot be stored this way.
+Storage and access to user data may be a factor in how you choose to configure authorization policies.
+
+NOTE: xref:ref-kafka-authorization-super-user-deployment-configuration-kafka[Super users] always have unconstrained access to a Kafka broker regardless of the authorization implemented on the Kafka broker.
+
+.Prerequisites
+
+* {ProductName} must be configured to use {oauth} with {oauth-server} for xref:assembly-oauth-authentication_str[token-based authentication].
+You use the same {oauth-server} server endpoint when you set up authorization.
+* You need to understand how to manage policies and permissions for {oauth-server} Authorization Services, as described in the {oauth-server-doc}.
+
+
+.Procedure
+
+. Access the {oauth-server} Admin Console or use the {oauth-server} Admin CLI to enable Authorization Services for the Kafka broker client you created when setting up {oauth} authentication.
+. Use Authorization Services to define resources, authorization scopes, policies, and permissions for the client.
+. Bind the permissions to users and clients by assigning them roles and groups.
+. Configure the Kafka brokers to use {oauth-server} authorization by updating the Kafka broker configuration (`Kafka.spec.kafka`) of your `Kafka` resource in an editor.
++
+[source,shell]
+----
+kubectl edit kafka my-cluster
+----
+
+. Configure the Kafka broker `kafka` configuration to use `keycloak` authorization, and to be able to access the authorization server and Authorization Services.
++
+For example:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka
+  # ...
+  authorization:
+    type: *keycloak* <1>
+    tokenEndpointUri: <__https://<auth-server-address>/auth/realms/external/protocol/openid-connect/token/introspect__> <2>
+    clientId: kafka <3>
+    delegateToKafkaAcls: false <4>
+    disableTlsHostnameVerification: false <5>
+    superUsers: <6>
+      - CN=fred
+      - sam
+      - CN=edward
+    tlsTrustedCertificates: <7>
+    - secretName: oauth-server-cert
+      certificate: ca.crt
+  #...
+----
+<1> Type `keycloak` enables {oauth-server} authorization.
+<2> URI of the {oauth-server} token introspection endpoint. For production, always use HTTPs.
+<3> The client ID of the {oauth} client definition in {oauth-server} that has Authorization Services enabled. Typically, `kafka` is used as the ID.
+<4> Delegate authorization to Kafka `SimpleACLAuthorizer` if access is denied by {oauth-server} Authorization Services policies.
+The default is `false`.
+<5> (Optional) Disable TLS hostname verification. Default is `false`.
```

Comment:
If this is optional, so is bullet no. 4, and no. 6.

Reply:
Added to _4_ and _6_